docs/modules/ROOT/pages/how-to/manage-projects-and-namespaces.adoc

```diff
@@ -0,0 +1,59 @@
+= Manage Projects and Namespaces
+
+This page explains the various mechanisms available to create OpenShift Projects in APPUiO Cloud.
+
+In Red Hat OpenShift, projects and namespaces can be used interchangeably, except when listing them. Check the xref:how-to/list-namespaces.adoc[corresponding page] for more information.
+
+== Creating Projects
+
+Through the web console:: Click on the btn:[Create Project] button and fill the form to create a new project.
+
+Through the Command Line:: Use the `oc new-project` command.
+
+Through the API:: Use the `ProjectRequest` resource to create a new project.
+
+== Project Ownership
+
+OpenShift Projects will always be assigned to the default organization specified by the `appuio.io/organization` label. Check the xref:explanation/organizations-and-rbac.adoc[Organizations and RBAC] page for more information.
+
+NOTE: If your user doesn't have a default organization, you must specify the label when creating the namespace or project.
+
+== Namespaces
+
+Namespaces can be created in two ways:
+
+. With the command line `kubectl create namespace` (or `oc create namespace`)
+. Through the API, using the `Namespace` resource.
+
+NOTE: You can't create namespaces through the web console.
+
+Namespaces created with any of these mechanisms can be managed just like any other OpenShift projects.
+
+Just like projects, namespaces are assigned to a specific organization using the `appuio.io/organization` label. Namespaces without such label will be assigned to the default organization. The code snippet below shows the YAML required to add the label to a new namespace request:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: My cool project
+    openshift.io/requester: my-username
+  labels:
+    appuio.io/organization: my-company # <1>
+    kubernetes.io/metadata.name: my-namespace
+    network-policies.syn.tools/no-defaults: 'true'
+    network-policies.syn.tools/purge-defaults: 'true'
```

Comment:
```suggestion
```

This should only be done when there is valid reason to do so. We never should add those to examples.